files/elb_lookup.py

```diff
@@ -1,60 +1,99 @@
 import os
 import boto3
 
-client = boto3.client('elb')
-clientv2 = boto3.client('elbv2')
-vpc_id = os.environ['VPC_ID']
-cluster_name = os.environ['CLUSTER_NAME']
+def _generate_lb_name_and_tags(client=None,
+                               clientv2=None):
+    ''' Generate tuples (LB name, tag description)
+    for both LB api versions, interleaving queries
+    between the v1 and v2 APIs.
+    '''
 
-def _get_elb_name():
+    vpc_id = os.environ['VPC_ID']
+
+    # Get the first page of results from both
+    # ELBv1 and ELBv2 APIs
     response = client.describe_load_balancers()
     response2 = clientv2.describe_load_balancers()
     load_balancers = response['LoadBalancerDescriptions']
     nlb_load_balancers = response2['LoadBalancers']
```

Comment:
first, we send an api request to get a list of load balancers. there are two different APIs for load balancers, so we ask for both of them. the catch that you discovered is that they will only send some, and not all (i.e. pagination)